notebooks/SMAP/3.0 SMAP Quality Flags.ipynb

```diff
@@ -0,0 +1,276 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    %matplotlib inline
</pre><p>This magic command should not be required in modern notebooks</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>